contributing-guides/style-guide.md

```diff
@@ -104,6 +104,10 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Description syntax
+
+- Use backticks for quoting in descriptions as they highlighted in GitHub.
```

Comment:
```suggestion
- Use backticks for quoting in descriptions as they're highlighted in GitHub.
```

```suggestion
- Use backticks for quoting in descriptions as they are highlighted in GitHub.
```

Your choice which!

Reply:
Might be even better if you include an example of what a backtick is, most of my friends don't know for example.

```suggestion
- Use backticks (\`) for quoting in descriptions as they're highlighted in GitHub.
```